test/ctia/test_helpers/store.clj

```diff
@@ -11,13 +11,26 @@
                    helpers/fixture-ctia
                    es-helpers/fixture-delete-store-indexes])})
 
-(s/defn test-for-each-store-with-app
+(s/defn test-selected-stores-with-app
   "Takes a 1-argument function which accepts a Trapperkeeper `app`
-  which should succeed for all stores."
-  [t :- (s/=> s/Any
+  which should succeed for the stores named in the first argument."
+  [selected-stores
+   t :- (s/=> s/Any
               (s/named s/Any 'app))]
-  (doseq [[store-key fixtures] store-fixtures]
+  (assert (set? selected-stores))
+  (doseq [:let [store-fixtures (select-keys store-fixtures selected-stores)
+                _ (assert (seq store-fixtures) "No stores selected")]
+          [store-key fixtures] store-fixtures]
     (testing (name store-key)
       (fixtures
         (fn []
           (t (helpers/get-current-app)))))))
+
+(s/defn test-for-each-store-with-app
+  "Takes a 1-argument function which accepts a Trapperkeeper `app`
+  which should succeed for all stores."
+  [t :- (s/=> s/Any
+              (s/named s/Any 'app))]
+  (test-selected-stores-with-app
+    (-> store-fixtures keys set)
```

Comment:
the two-space indentation, combined with the newline before the first arg makes it look as if `test-selected-stores-with-app` was a macro that took an arbitrary `& body`

Which makes `(-> store-fixtures keys set)` look like dead code :)

The following makes it more evident that we're looking at a vanilla function call:

```clj
(test-selected-stores-with-app (-> store-fixtures keys set)
                               t)
```